content/300-guides/200-deployment/110-deployment-guides/001-deploying-to-aws-lambda.mdx

```diff
@@ -0,0 +1,305 @@
+
+</Admonition>
+
+## 3. Create a Prisma schema from your database
+
+`prisma db pull` takes the tables and relations that you define in your database and pulls them into a `schema.prisma` file. In the root of your project, run:
+
+```terminal
+npx prisma db pull
+```
+
+This command also generates Prisma Client based on your Prisma schema. After your `schema.prisma` file has been created, be sure to [add the correct `binaryTarget` to your schema.](#binary-targets-in-schema.prisma)
+
+## 4. Baseline your database
+
+After your schema has been created, be sure to [baseline your database](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/baseline-your-database-typescript-postgresql) in order to let Prisma know that it should not try to migrate anything (yet!)
+
+From your project root (or the directory that contains your `prisma/` directory) run
+
+```terminal
+mkdir -p prisma/migrations/0_init
+npx prisma migrate diff --from-empty --to-schema-datamodel prisma/schema.prisma --script > prisma/migrations/0_init/migration.sql
+```
+
+These commands will create an initial, empty migration file and then populate it with the current structure of your database. After you've confirm that the `0_init` file looks good, mark it as applied with the command:
+
+```terminal
+npx prisma migrate resolve --applied 0_init
+```
+
+### Result
+
+You should now have a `schema.prisma` and Prisma Client that match your existing database schema. Prisma and your database will also be in sync.
+
+## 4. Set your AWS Access key as an environment variable
+
+In order for the Serverless Framework to provision the AWS resources and deploy your application, you must configure your access key.
+
+The method you use to get the access key depends on whether you use your personal account or create a special IAM user for the Serverless Framework. We recommend that you create a special IAM user because it lets you set granular permissions. To get an access key for your account, follow the [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey)
+
+When you have the _Access key ID_ and a _Secret access key_, set them with the following command. In the following command, replace the placeholders with your _Access key ID_ and _Secret access key_.
+
+Use the following command:
+
+```terminal wrap
+serverless config credentials --provider aws --key AWS_ACCESS_KEY_ID  --secret AWS_SECRET_ACCESS_KEY
+```
+
+### Result
+
+If the configuration is successful, you see the following result in the terminal.
+
+```terminal
+✔ Profile "default" has been configured
+```
+
+## 5. Deploy and test your project
+
+You have finished Prisma's configuration, so you can now deploy your application as normal.
+
+```terminal
+serverless deploy
+```
+
+## Things to know
```

Comment:
The content contained here is what I would consider valuable for the deployment guide.

We can also split the "Deploying to AWS" into multiple sections depending on the tooling used, i.e.,
- Using Serverless Framework
- Using SST
- [Insert tooling we don't know about yet]